server/shared/types/routes/gdc.maf.ts

```diff
@@ -15,6 +15,18 @@ export type File = {
 	project_id: string
 }
 
+enum ExperimentalStrategy {
+	targeted = 'Targeted Sequencing',
+	wxs = 'WXS'
+}
+
+export type GdcMafRequest = {
+	/** Name of exp strategy to get maf files for */
+	experimentalStrategy: ExperimentalStrategy
```

Comment:
enum check doesn't work. try changing line 35 of client/src/gdc.maf.js to pass an invalid value, no alert from ts

Reply:
this error should also be caught by server/routes/, to guard against attacks. can this be done?

Reply:
I need to look into a little more if that's the goal. `Enum` isn't standardized typescript. From what I quickly read earlier, `enum` is checked at runtime, not compile time. Also, I wrote in my class notes that enums can pass incorrect values without an error. So with my introductory knowledge on enums, if you want to use this method, it will be trickier to catch errors.

Reply:
@xzhou82 I rebased on master to avoid seeing unrelated errors that were already fixed today. If you now run `npx tsc`, there should not be any type check errors with my fixes.

For checking request parameters/body during run time, you can `import { validGdcMafRequest } from '../shared/checkers/index.js'`, that's the automatically generated validation function.

Reply:
thanks can merge now